gidgethub/test/test_sansio.py

```diff
@@ -310,15 +310,18 @@ def test_403_forbidden(self):
 
     def test_422(self):
         status_code = 422
-        errors = [{"resource": "Issue", "field": "title",
-                   "code": "missing_field"}]
+        errors = [
```

Comment:
Please don't remove the preexisting test, instead making a new test for this new case as the old case is still valid.